amr.py

```diff
@@ -203,6 +203,23 @@ def parse_AMR_line(line):
         # Last significant symbol is / --- start processing node value (concept name)
         # Last significant symbol is ) --- current node processing is complete
         # Note that if these symbols are inside parenthesis, they are not significant symbols.
+        
+        exceptions =set(["prep-on-behalf-of", "prep-out-of", "consist-of", "subset-of"])
```

Comment:
```suggestion
        exceptions =set(["prep-on-behalf-of", "prep-out-of", "consist-of"])
```

`:subset` and `:subset-of` actually are inverses, though these roles have been abandoned in favor of `include-91` (except for a few erroneous annotations: https://github.com/amrisi/amr-guidelines/issues/236).

Reply:
ok. I see. FYI, what I have done is enumerating all relations in AMR training split (of LDC2017T10), and find all relations with the form "X-of" but "X" itself cannot be found in the corpus. They are "prep-on-behalf-of", "prep-out-of", "consist-of" and "subset-of".

I have modified the PR with all your suggestions.